integration-tests/helpers/fake_agent.js

```diff
@@ -0,0 +1,257 @@
+'use strict'
```

Comment:
nearly every file in our repo uses a hyphen `-` as a separator in the name rather than underscores `_`. Lets keep that consistency.

Reply:
folders ? yes. Files ? eeehhhh ? this is the dd-trace/src/ folder:
![image](https://github.com/user-attachments/assets/694c2948-54b2-4166-803f-ef195b5831ca)

Reply:
![image](https://github.com/user-attachments/assets/f3d3c9ac-f22a-4d22-8d01-8dff7e218cff)

Reply:
now it does look like integration-tests files have a tendency to use hyphen instead of underscore :+1:

Reply:
I guess that's why I got confused and used the underscore. In any case, I've now reamed the file to `fake-agent.js` (in Kibana we had a lint rule for our filenames 😉)